contracts/Token.sol

```diff
@@ -15,34 +15,26 @@
   along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
 */
 
-pragma solidity ^0.4.23;
-pragma experimental "v0.5.0";
+pragma solidity >=0.5.3;
 
+import "../lib/dappsys/token.sol";
 
-import "../lib/dappsys/auth.sol";
-import "../lib/dappsys/base.sol";
-import "./ERC20Extended.sol";
-
-
-contract Token is DSTokenBase(0), DSAuth, ERC20Extended {
-  uint8 public decimals;
-  string public symbol;
-  string public name;
 
+contract Token is DSToken {
```

Comment:
DSToken is DSStop - should we be considering using that?

Reply:
So the entire recovery functionality we have in the Network and Colony contracts is not applicable to Token contract here. We might as well use the [built in logic](https://github.com/dapphub/ds-stop)

Reply:
[There we go](https://github.com/JoinColony/colonyToken/pull/14/commits/2adc605ac330ff38116ee6ecfd37e8aab6025c00)

Reply:
I've gone back a little on design here and chosen to reuse our original implementation of Token with the addition of upgrading the contract to Solidity 0.5. and adding overloads for `mint` and `burn`.